docs/data-sources/project.md

```diff
@@ -10,11 +10,19 @@ data "gitlab_project" "example" {
 }
 ```
 
+```hcl
+data "gitlab_project" "example" {
+	id = "foo/bar/baz"
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
-* `id` - (Required) The integer that uniquely identifies the project within the gitlab install.
+* `id` - (Required) The integer or path with namespace that uniquely identifies the project within the gitlab install.
+
+**Note**: Exactly one of `id` or `full_path` must be provided.
```

Comment:
Can you revert this note please

Reply:
Yup, sorry I missed that. 091c23f should do it.